front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -155,14 +155,13 @@ export function CommandMenu() {
                 to={`person/${person.id}`}
                 label={person.displayName}
                 key={person.id}
-                icon={
-                  <Avatar
-                    avatarUrl={null}
-                    size="sm"
-                    colorId={person.id}
-                    placeholder={person.displayName}
-                  />
-                }
+                Icon={Avatar}
```

Comment:
Ok nice.

We should close the API to receive only : `IconComponent | AvatarProps` because they are not the same, and we want only one or the other, not two props. 

This will require a typeguard in CommandMenuItem, see what you can do, but here it should be separated, because Avatar is not an Icon.